doc-content/enterprise-only/springboot/openshift-springboot-proc.adoc

```diff
@@ -0,0 +1,123 @@
+[id='openshift-springboot-proc_{context}']
+= Running a SpringBoot business application on {OPENSHIFT}
+
+To run your {PRODUCT} SpringBoot business application on {OPENSHIFT}, create an immutable image and push this image to your {OPENSHIFT} environment.
+
+ifdef::PAM[]
+Optionally, you can also use {CENTRAL} Monitoring to monitor the execution of business processes in your application.
+endif::PAM[]
+
+.Prerequisites
+
+* You developed a {PRODUCT} SpringBoot business application. For instructions about creating the application, see xref:bus-app-create_{context}[].
+* If necessary, you configured Spring security for the application. For instructions about configuring Spring security, see xref:bus-app-security-con_{context}[].
+* You completed other Spring configuration for the business application, as necessary. For instructions about Spring configuration for your business application, see xref:bus-app-configure-con_{context}[].
```

Comment:
This sentence does not read well. "You completed other Spring configuration for the business application, as necessary."

Maybe change to:
You have completed any additional Spring configuration for the business application.